modules/npm/src/main/scala/lucuma/core/exports.scala

```diff
@@ -3,27 +3,31 @@
 
 package lucuma.core
 
-import scala.scalajs.js
 import cats.syntax.all.*
-import scala.scalajs.js.annotation.JSExportTopLevel
 import lucuma.core.enums.Site
-import java.time.Instant
-import lucuma.core.model.CoordinatesAtVizTime
-import java.time.Duration
+import lucuma.core.enums.TwilightType
+import lucuma.core.math.Angle
 import lucuma.core.math.Coordinates
-import lucuma.core.math.skycalc.SkyCalcResults
+import lucuma.core.math.HourAngle
+import lucuma.core.math.parser.AngleParsers
 import lucuma.core.math.skycalc.ImprovedSkyCalc
+import lucuma.core.math.skycalc.SkyCalcResults
+import lucuma.core.model.CoordinatesAtVizTime
 import lucuma.core.model.ObservingNight
+
+import java.time.Duration
+import java.time.Instant
 import java.time.LocalDate
-import lucuma.core.enums.TwilightType
+import scala.scalajs.js
 import scala.scalajs.js.JSConverters.*
+import scala.scalajs.js.annotation.JSExportTopLevel
 
 /**
  * JS facade to plot points calculated with the method `nightPlot`.
  */
 @JSExportTopLevel("PlotPoint")
 class PlotPoint(val instant: js.Date, val airmass: Double, val altitude: Double) extends js.Object {
```

Comment:
@cquiroz Is this used anywhere? If it can be removed I'd like to. It adds about 350kB to the bundle size for navigate that cannot be tree-shaked away

Reply:
@stroncod Are you using this?

Reply:
We are not currently using it in the Scheduler so you can go ahead and remove it if you want.